### ISSUE TITLE
DryIoc Container RegisterForNavigation<NavigationPage>() causes exception with 7.0.0.336-pre

### ISSUE THREAD

**Issue by therealjohn** [PrismLibrary/Prism#1309]
After updating to the latest pre from 6.3, I am unable to register a `NavigationPage`. This looks similar to [#1304](https://github.com/PrismLibrary/Prism/issues/1304).

### Package info

- Platform: iOS/Android
- Prism version: 7.0.0.336-pre
- Xamarin version (if applicable): XF 2.5.0.121934

### Repro steps

```
protected override void RegisterTypes(IContainerRegistry containerRegistry)
{
       containerRegistry.RegisterForNavigation<NavigationPage>();
}
```

### Actual Results

```
DryIoc.ContainerException:
Unable to select single public constructor from implementation type Xamarin.Forms.NavigationPage:
Void .ctor(), 
Void .ctor(Page)
```

### Excepted Results

Successfully register a `NavigationPage` for navigation.

**Comment by dansiegel** [comment]
This is a duplicate of #1304 and has been resolved in #1305. Please use the latest CI package on MyGet.

**Comment by therealjohn** [comment]
Thanks for confirming, Dan!
